config/helm/chart/default/values.yaml

```
@@ -71,7 +77,14 @@ csidriver:
       operator: Exists
   labels: []
   annotations: []
+  csi_init:
```

```suggestion
  csiInit:
```
let's stick with camelCase like in other places of the `values.yaml`, maybe it even makes sense to change the init container name